components/CreateOrganizationForm.js

```diff
@@ -101,9 +102,8 @@ const CreateOrganizationForm = props => {
     if (values.description.length > 150) {
       errors.description = intl.formatMessage(orgMessages.errorDescription);
     }
-    const regexExp = /[-a-zA-Z0-9@:%._/+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_/+.~#?&//=]*)?/gi;
 
-    if ((values.website && !values.website.match(new RegExp(regexExp))) || values.website.startsWith('http')) {
+    if (values.website && !isURL(values.website)) {
```

Comment:
It seems `values.website` will be without the scheme part (`https://`), is this still working?

Reply:
Thanks for the feedback, I still needed to test the PR before putting it under review. I'll fix that and add a type on the input, as mentioned in https://github.com/opencollective/opencollective/issues/4706#issuecomment-922703523.